### ISSUE TITLE
Publicize: Better announce secondary reconnection is necessary

### ISSUE THREAD

**Issue by kraftbj** [Automattic/jetpack#1684]
Problem: When a secondary JP user (e.g. user whose linked their account to WP.com on a site already connected by another user) has added Publicize connections, then the master connection is cycled, those connections are still present even though the user's link was severed.

Steps to reproduce:
1. User "A" connects a JP site to his/her WP.com account.
2. User "B" connects their JP user account to their separate WP.com account.
3. User B adds various Publicize connections.
4. User A disconnects the site from WP.com and reconnects.
5. User B's posts won't go out to their Pub accounts
6. User B visits Settings->Sharing and see all of their connections.
7. User B visits the Jetpack page and see prompt to connect their account with WP.com.

In short, we should either make a reconnection prompt more obvious and/or hide the Publicize connections when User B is in this disassociated state.

@csonnek
